ml-proto/host/import/env.ml

```diff
@@ -0,0 +1,39 @@
+open Values
+open Types
+
```

Comment:
Added comments to `spectest` and `env` modules explaining their purpose. PTAL

@sunfishcode, as explained before, the `env` module emulates the respective module imported by Binaryen code. Renaming it here would defeat the purpose. Binaryen would have to decide that.